src/DonateHope.Core/ServiceContracts/CampaignRatingsServiceContracts/ICampaignRatingCreatingService.cs

```diff
@@ -0,0 +1,13 @@
+using DonateHope.Core.DTOs.CampaignContributionDTOs;
+using DonateHope.Core.DTOs.CampaignRatingDTOs;
+using FluentResults;
+
+namespace DonateHope.Core.ServiceContracts.CampaignRatingsServiceContracts;
+
+public interface ICampaignRatingCreatingService
```

Comment:
I did a bit of research about naming services - for this case, people prefer: `ICampaignRatingCreateService` over `ICampaignRatingCreatingService`. Let's follow the community @cungbac

Reply:
Also, apply the same to other service. Such as `RetrievalService` -> `RetrieveService`